app/src/main/java/org/isoron/uhabits/io/HabitsCSVExporter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2016 Álinson Santos Xavier <isoron@gmail.com>
+ * Copyright (C) 2016 Γlinson Santos Xavier <isoron@gmail.com>
```

Comment:
This is probably an encoding error. Please revert this line to before and make sure that your text editor is configured to use UTF-8. The same happened in every file.